project/AutomaticModuleName.scala

```diff
@@ -9,12 +9,13 @@ import sbt.{ Def, _ }
  *
  * !! DO NOT BE TEMPTED INTO AUTOMATICALLY DERIVING THE NAMES FROM PROJECT NAMES !!
  *
- * The names carry a lot of implications and DO NOT have to always align 1:1 with the group ids or package names,
- * though there should be of course a strong relationship between them.
+ * The names carry a lot of implications and DO NOT have to always align 1:1 with the group ids or package names, though
+ * there should be of course a strong relationship between them.
  */
 object AutomaticModuleName {
   private val AutomaticModuleName = "Automatic-Module-Name"
 
-  def settings(name: String): Seq[Def.Setting[Task[Seq[PackageOption]]]] =
-    Seq(packageOptions in (Compile, packageBin) += Package.ManifestAttributes(AutomaticModuleName -> name))
```

Comment:
The correct migration would be (see https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#slash)
```
Compile / packageBin / packageOptions += Package.ManifestAttributes(AutomaticModuleName -> name)
```

But anyways I think this `AutomaticModuleName` is not used anywhere. Could be removed in another fix.